src/duplicate_count.py

```diff
@@ -0,0 +1,7 @@
+def duplicate_count(string):
+    char_count = {
+        char.lower()
+        for char in string
+        if string.lower().count(char.lower()) > 1 and char.isalnum()
+    }
```

Comment:
This is very in efficient.  For every character in the string it has to count, even if it has already counted.  It is an interesting use of a "set comprehension".

Better would be to do the counting yourself.

```python
def duplicate_count(string):
    counts = {}
    for char in string:
        if char.isalnum():
            counts[char] = counts.get(char, 0) + 1
    return sum(1 for _ in counts.values() if i > 1)
```